fhir-persistence-jdbc/src/test/java/com/ibm/fhir/persistence/jdbc/search/test/JDBCSearchNearTest.java

```diff
@@ -0,0 +1,296 @@
+/*
+ * (C) Copyright IBM Corp. 2019
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.jdbc.search.test;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertNotEquals;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertNull;
+import static org.testng.AssertJUnit.assertTrue;
+
+import java.io.InputStream;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.logging.LogManager;
+
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import com.ibm.fhir.config.FHIRConfiguration;
+import com.ibm.fhir.config.FHIRRequestContext;
+import com.ibm.fhir.model.resource.Location;
+import com.ibm.fhir.model.resource.Resource;
+import com.ibm.fhir.model.test.TestUtil;
+import com.ibm.fhir.persistence.FHIRPersistence;
+import com.ibm.fhir.persistence.MultiResourceResult;
+import com.ibm.fhir.persistence.SingleResourceResult;
+import com.ibm.fhir.persistence.context.FHIRPersistenceContext;
+import com.ibm.fhir.persistence.context.FHIRPersistenceContextFactory;
+import com.ibm.fhir.persistence.exception.FHIRPersistenceException;
+import com.ibm.fhir.persistence.jdbc.impl.FHIRPersistenceJDBCImpl;
+import com.ibm.fhir.persistence.jdbc.test.util.DerbyInitializer;
+import com.ibm.fhir.search.context.FHIRSearchContext;
+import com.ibm.fhir.search.util.SearchUtil;
+
+/**
+ * <a href="https://www.hl7.org/fhir/r4/location.html#positional">FHIR
+ * Specification: Positional Searching for Location Resource</a>
+ * <br>
+ * Original LONG/LAT
+ * 
+ * <pre>
+ *  "position": {
+ *      "longitude": -83.6945691,
+ *      "latitude": 42.25475478,
+ *      "altitude": 0
+ * }
+ * </pre>
+ */
+public class JDBCSearchNearTest {
+    private Properties testProps;
+
+    protected Location savedResource;
+
+    protected static FHIRPersistence persistence = null;
+
+    @BeforeClass
+    public void startup() throws Exception {
+        InputStream inputStream = JDBCSearchNearTest.class.getResourceAsStream("/near.unitTest.properties");
```

Comment:
does it need its own unitTest properties file?  why not just re-use `logging.unitTest.properties`?

Reply:
I turn on com.ibm.fhir.search trace with this one.

Reply:
also exists in fhir-persistence-jdbc

Reply:
yeah, i guess i was just thinking you could add a line for `com.ibm.fhir.search` in the existing `logging.unitTest.properties` file.  both will be INFO by default and you can selectively change the log level.  i don't see why we need a separate file for it, but i guess its not a big deal.

Reply:
right.  just explaining my choice.  I can reconcile with the existing one

Reply:
Reconciled.